api/data_explorer/util/elasticsearch_util.py

```diff
@@ -185,3 +188,52 @@ def get_elasticsearch_facet(es, elasticsearch_field_name, field_type):
         es_facet = NestedFacet('samples', es_facet)
 
     return es_facet
+
+
+def _delete_index(es, index):
+    current_app.logger.info('Deleting %s index.' % index)
+    try:
+        es.indices.delete(index=index)
+    except Exception as e:
+        # Sometimes delete fails even though index exists. Add logging to help
+        # debug.
+        current_app.logger.info('Deleting %s index failed: %s' % (index, e))
+        # Ignore 404: index not found
+        index = es.indices.get(index=index, ignore=404)
+        current_app.logger.info(
+            'es.indices.get(index=%s): %s' % (index, index))
+
+
+def _create_index(es, index, mappings_file=None):
+    current_app.logger.info('Creating %s index.' % index)
```

Comment:
opt Delete this logger.info, add a logger.info to load_index() "Loading % index from JSON cache"